backend/mora/service/detail_reading.py

```diff
@@ -163,6 +163,8 @@ def get_detail(type, id, function):
     :<jsonarr string validity: The validity times of the object.
     :<jsonarr boolean primary: Whether this is the one and only main
                                position for the relevant person.
+    :<jsonarr string fraction: An indication of how much this
```

Comment:
`int`? Also; perhaps explain what the value actually means; i.e. what does 42 mean? 42 horses or perhaps percent?